chia/server/upnp.py

```diff
@@ -7,16 +7,33 @@
 
 
 log = logging.getLogger(__name__)
+try:
+    upnp = miniupnpc.UPnP()
+    upnp.discoverdelay = 30
+    upnp.discover()
+    upnp.selectigd()
```

Comment:
are these blocking calls? If so, I really think we need to do this in a separate thread (or put `await` in front of them). This being the global context won't run in a coroutine though, so I would expect this would have to be moved back.